multimodal/src/autogluon/multimodal/models/utils.py

```diff
@@ -558,10 +558,10 @@ def get_hf_config_and_model(
     -------
     A Huggingface config and model.
     """
-    config = AutoConfig.from_pretrained(checkpoint_name)
+    config = AutoConfig.from_pretrained(checkpoint_name, torchscript=True)
 
     if pretrained:
-        model = AutoModel.from_pretrained(checkpoint_name, low_cpu_mem_usage=low_cpu_mem_usage)
+        model = AutoModel.from_pretrained(checkpoint_name, low_cpu_mem_usage=low_cpu_mem_usage, torchscript=True)
```

Comment:
Setting `torchscript=True` make all the tied weights in model decoupled, which can cause problems if finetuning the model. See https://huggingface.co/transformers/v3.0.2/torchscript.html#torchscript-flag-and-tied-weights.

Reply:
Here is a list of things to assess the potential impact of this change.
1. see how many models are impacted by tied weights, especially hf_text and derived models.
2. evaulate the potential accuracy loss when decouple the layers that have tied weights
3. try to work around the torchscript argument, and try to avoid setting it to True, in order to avoid the issue
4. (less likely but possible) if the impact of tied weight is high, and impact of TRT is relatively slow, while we don't have workaround to avoid this issue, we could potentially abandon the idea of introducing the onnx/trt exporting feature.

Reply:
Since onnx/trt is mainly for inference, how about still using the previous logic that scripts the model immediately before exporting onnx? Setting `torchscript=True` here can impact training, which may require tied weights, especially for the generative models to add.

Reply:
> how about still using the previous logic that scripts the model immediately before exporting onnx?

Yes, this is a better solution to satisfy both worlds. I will definitely give it a try.

Reply:
torchscript=True argument has been removed, and seems everything is still working without the argument.